### ISSUE TITLE
CDD11 traindataset and testdataset

### ISSUE THREAD

**Issue by aodwals** [eduardzamfir/MoCE-IR#14]
How are the CDD11 train dataset and test dataset set up? Thanks very much

**Comment by aodwals** [comment]
Hello, I just want to confirm whether the training set was randomly selected from the entire CDD11 dataset, and whether the test set is the entire CDD11 dataset. Thank you to the authors for providing the opportunity for learning and discussion; we will cite your paper.

**Comment by eduardzamfir** [comment]
Depending on the task configuration, we select random images from the CDD11 dataset and also test only on that subset. We also ran one experiment over all possible degradations. Please check the descriptions in the paper for more info.
